src/network.js

```diff
@@ -130,10 +134,10 @@ class Network {
   async sendMessage (peer, msg) {
     if (!this._running) throw new Error('network isn\'t running')
 
-    const stringId = peer.toB58String() ? peer.toB58String() : peer.id.toB58String()
+    const stringId = peer.toB58String()
     this._log('sendMessage to %s', stringId, msg)
 
-    const { conn, protocol } = await this._dialPeer(peer)
+    const { stream, protocol } = await this._dialPeer(peer)
```

Comment:
Does this need a try/catch?

Reply:
It shouldn't need one because `sendMessage` already throws. There are catches in both places sendMessage is used.